Source/Resources/Content/SharePoint/TenantProperties/Configuration.json

```diff
@@ -15,7 +15,6 @@
   "BlockDownloadOfAllFilesOnUnmanagedDevices": false,
   "BlockDownloadOfViewableFilesForGuests": false,
   "BlockDownloadOfViewableFilesOnUnmanagedDevices": false,
-  "BlockMacSync": false,
```

Comment:
I was thinking we'd remove these in a separate PR after I merge changes into M365Management